tests/integration/conftest.py

```diff
@@ -8,10 +8,11 @@
 
 import pytest
 from fastapi.testclient import TestClient
+from sqlalchemy import text
 from sqlalchemy.orm import scoped_session
 from sqlalchemy.orm import sessionmaker
 
-from apibox.application.cli.commands.database import config
+from apibox.application.cli.commands.database import config as database_config
```

Comment:
had to rename it to comply with pylint (the `config` object was being obfuscated in a function)